pmlc/rt/runtime_registry.cc

```diff
@@ -13,20 +14,17 @@
 namespace pmlc::rt {
 namespace {
 
-struct LoaderList {
-  std::mutex mutex;
-  std::list<std::pair<std::string, Loader>> loaders;
-
-  static LoaderList &instance() {
-    static LoaderList ll;
-    return ll;
-  }
+class DuplicateRuntimeError final : public std::runtime_error {
```

Comment:
So far we haven't been using custom exceptions, is `std::runtime_error` insufficient?

Reply:
Done.